### DISCUSSION TITLE
Archive/Close a discussion

### DISCUSSION THREAD

**Discussion by pavlenex** [community/community#3244] [Discussions] [answered]
Sometimes a discussion shouldn't be in the main view. For example a repetitive support question or something that's already taking place in another thread. In that case, like with issues, it would be helpful to hide/archive/close a discussion without deleting it or leaving it opened to spam the main board view.

**Comment by HDVinnie** [comment]
Totally agree. Something to mark a discussion as complete and hide it from default view. Then can see all completed discussions via a filter.

**Reply by senorflor** [reply]
We have done this with locking, though not perfect: https://github.com/github/feedback/discussions/3568

**Reply by senorflor** [reply]
On the right bar:

<img width="344" alt="Screen Shot 2021-03-17 at 9 22 32 AM" src="https://user-images.githubusercontent.com/4453806/111485590-bdde1800-8704-11eb-8d06-c934695a4ae2.png">

Then you can filter with `is:unlocked`

**Reply by theilgaz** [reply]
Locking is not hiding from timeline or category page. We should have an option to hide all "done" discussions from everywhere at once. Some issues are including many features, bugs and opinions. Discussions are great to handle multiple issues at once but when all issues completed, we have to hide discussion.

**Reply by azinit** [reply]
> Discussions are great to handle multiple issues at once but when all issues completed, we have to hide discussion.

Totally agree with you!

We have literally same problem and at the moment - **there are a lot of idle discussions, that couldn't be resolved and hidden from general list :(**

<details>

<summary><i>But unfortunately - now we should manually change category, title and discussion's description...</i></summary>

![image](https://user-images.githubusercontent.com/42924400/111950466-5bfe2380-8af3-11eb-8251-15260aea5e5c.png)

![image](https://user-images.githubusercontent.com/42924400/111950563-84861d80-8af3-11eb-9bc1-e16a9247a911.png)

</details>

**Reply by senorflor** [reply]
Yes, it's an imperfect workaround, as mentioned in the first message: you have to include the `is:unlocked` in the search bar. Only commented here in case it helped you with a slightly less laborious workaround while we wait for decisions/progress on the feature. The search parameter does reflect in the URL, so you can link to the unlocked-only views, but still not perfect, as navigation between categories sometimes removes the `is:unlocked` (e.g. from a particular category to view all).

**Reply by senorflor** [reply]
Also did not consider timeline page: good call for those that use that!

**Reply by theilgaz** [reply]
> Yes, it's an imperfect workaround, as mentioned in the first message: you have to include the `is:unlocked` in the search bar. Only commented here in case it helped you with a slightly less laborious workaround while we wait for decisions/progress on the feature. The search parameter does reflect in the URL, so you can link to the unlocked-only views, but still not perfect, as navigation between categories sometimes removes the `is:unlocked` (e.g. from a particular category to view all).

It can be a temp solution for a moment but when you navigate to somewhere else, that will be removed which means we can't set our default or pre-defined filter. Closing is not a new thing for GitHub, all we are using issues for a long time and most of GitHub users will love closing discussions like issues.

**Comment by PranamBhat** [comment]
Hi @pavlenex 

Agree. Thank you for the feedback.

@evi-liu @candyho Can you please have a look at this earliest. Thank you.

**Comment by azinit** [comment]
Still actual! Looks as redundant in common list of unlocked discussions!

**Comment by serefilgaz** [comment]
This would be perfect.

**Comment by WrichikBasu** [comment]
Seriously need an option to archive discussions. My repo has a number of discussions that are no longer active, but deleting them off is not an option as I would like to preserve history. Hiding/archiving them is the only way to go.

**Comment by Relequestual** [comment]
Consider also upvoting https://github.com/github/feedback/discussions/3097 which at current has more votes

**Comment by abhinavvadrevu** [comment, marked as answer]
Hi all, we are really excited to introduce the ability to close Discussions! Please read more about the feature here: https://github.blog/changelog/2023-03-14-adding-the-ability-to-close-github-discussions/

**Reply by WrichikBasu** [reply]
That's great!

**Reply by tony-sol** [reply]
That's great, but how to close discussion in archived project? There is no close button 😭
